src/index.html

```diff
@@ -8,24 +8,21 @@
     <h1>
       Transforming technology within the <br> Department of Defense
     </h1>
-    <p class="subtitle">
-      An agency team of the <a href="https://www.usds.gov">U.S. Digital Service</a>, using <br> design and technology to improve the lives of American citizens.
-    </p>
   </div>
 </section>
 
 <section id="about" class="background-statement usa-grid">
   <div class="usa-width-one-whole">
-    <h2>By the people. For the people.</h2>
+    <h2>Solving Big Problems. Supporting Critical Missions.</h2>
   </div>
 
   <div class="usa-width-one-whole">
     <div class="page-width-70">
       <p>
-        The Department of Defense defends and protects our nation's military and civilians across the globe. Our projects range widely, from strengthening our national security, to taking care of our Veterans, or deploying next-generation GPS for billions of people. However, oftentimes our progress is hindered by outdated tools and practices that lag far behind private sector standards.
+        The Department of Defense defends and protects our nation's military and civilians across the globe. However, oftentimes progress may be hindered by outdated systems and technology.
       </p>
       <p>
-        Our mission is to drive a giant leap forward in the way DOD builds and deploys technology and digital services. We work alongside our public servants and service members, empowering them to incorporate private sector best practices and talent to build a better <s>future</s> now.
+        The Defense Digital Service is an agency team of the <a href="https://www.usds.gov">U.S. Digital Service</a>. Our mission is to drive a giant leap forward in the way the agency builds, buys and deploys technology and digital services. We hire top technologists for short-term "tours of civic duty" to work alongside civil servants and service members building better tools to strengthen national security and care for military members and their families.
```

Comment:
@MoVaheyDDS I decided to remove the subtitle text in the header image instead of changing it to what you requested because the background made it very hard to read. I tried some other images and I think it's cleaner to just get rid of the subtitle text and merge it with this. Also they both said "Our mission is X" where X was different. Let me know if you want me to change this copy.

Reply:
Mo isn't on GitHub much (like... at all). I would send her the preview link and get her sign off. There is one other comment below from me as well.

https://deploy-preview-28--dds-mil.netlify.com/